src/plugins/expressions/common/execution/execution.ts

```diff
@@ -186,13 +199,13 @@ export class Execution<
     });
 
     const inspectorAdapters =
-      execution.params.inspectorAdapters || createDefaultInspectorAdapters();
+      (execution.params.inspectorAdapters as InspectorAdapters) || createDefaultInspectorAdapters();
```

Comment:
why passed in type is not correct ? should probably be fixed

Reply:
That's because the `params.inspectorAdapters` is a supertype of the `InspectorAdapters`, but the execution context requires the `InspectorAdapters` as from the type parameters. That's a legit solution and definitely not a hack.